### PR TITLE
Remove WorkerHearbeat message and cleanup proto file

### PR DESCRIPTION
**Pull request by liliankasem**
Related to https://github.com/Azure/azure-functions-host/issues/8164

### CODE REVIEW

File: src/proto/FunctionRpc.proto [hunk 1]
```diff
@@ -144,10 +141,7 @@ message StatusResult {
   repeated RpcLog logs = 3;
 }
 
-// TODO: investigate grpc heartbeat - don't limit to grpc implemention
-
-// Message is empty by design - Will add more fields in future if needed
-message WorkerHeartbeat {}
+// TODO: investigate grpc heartbeat - don't limit to grpc implementation
```
Comment (madelinegordon) [inline src/proto/FunctionRpc.proto:144]: nit: Is this todo still needed?

File: src/proto/FunctionRpc.proto [hunk 2]
```diff
@@ -179,24 +173,25 @@ message FileChangeEventRequest {
...
 
-// NOT USED
-message WorkerStatusRequest{
+// Used by the host to determine worker health
+message WorkerStatusRequest {
 }
 
-// NOT USED
+// Worker responds with status message
+// TODO: Add any worker relevant status to response
```
Comment (madelinegordon) [inline src/proto/FunctionRpc.proto:194]: same as above

**Review by madelinegordon — APPROVED** [review]

File: src/proto/FunctionRpc.proto (hunk 1 above)
Comment (liliankasem) [reply src/proto/FunctionRpc.proto:144]: Hmmm not sure, this was added 4 years ago, maybe there is a different user story for it?

File: src/proto/FunctionRpc.proto (hunk 2 above)
Comment (liliankasem) [reply src/proto/FunctionRpc.proto:194]: still relevant, this todo is part of the dynamic concurrency feature

File: src/proto/FunctionRpc.proto (hunk 1 above)
Comment (liliankasem) [reply src/proto/FunctionRpc.proto]: didn't find anything to track this, might not be relevant 4 years later so will remove unless some says not to :)

**Review by ejizba — APPROVED** [review]

**Pull request merged**